cartographer_ros/cartographer_ros/node.h

```diff
@@ -152,7 +152,7 @@ class Node {
   void PublishSubmapList(const ::ros::WallTimerEvent& timer_event);
   void AddExtrapolator(int trajectory_id, const TrajectoryOptions& options);
   void AddSensorSamplers(int trajectory_id, const TrajectoryOptions& options);
-  void PublishTrajectoryStates(const ::ros::TimerEvent& timer_event);
+  void PublishLocalTrajectoryData(const ::ros::TimerEvent& timer_event);
```

Comment:
Reading this, I prefer "PublishTrajectoryData". It publishes the entire trajectory, the only thing local about it is it also looks at LocalSlamResult data to show that trajectory.

Reply:
It publishes the scan matched pointcloud and the TF.

I think you mean `Node::PublishTrajectoryNodeList` which is not touched by this change?

Reply:
Sorry, you are right, I was mistaken about what PublishTrajectoryStates does.
Since there are no more open comments, should I merge?

Reply:
No problem, merging is fine for me.